po/es.po

```diff
@@ -81,158 +80,150 @@ msgstr "_Cancelar"
 #: src/accountdialog.ui:22 src/remotedesktopdialog.ui:28
 #: src/screencastdialog.ui:28 src/screenshotdialog.ui:38
 msgid "_Share"
-msgstr "_Compartir"
+msgstr "Co_mpartir"
 
 #: src/accountdialog.ui:148
 msgid "Make changes before sharing the information"
-msgstr ""
+msgstr "Realizar cambios antes de compartir la información"
 
 #: src/appchooserdialog.c:154
 msgid "Failed to start Software"
-msgstr ""
+msgstr "No se pudo iniciar Software"
 
 #: src/appchooserdialog.c:373
 #, c-format
 msgid ""
 "Select an application to open “%s”. More applications are available in <a "
 "href='software'>Software.</a>"
 msgstr ""
+"Seleccione una aplicación para abrir «%s». Hay más aplicaciones "
```

Comment:
Why the `« »` here? Double quotes should be fine in Spanish

Reply:
The guillemets are part of a standardized style that’s used across GNOME, KDE, Ubuntu and LibreOffice.

Reply:
Oh! I see... I'm no translator (I should have made a disclaimer on that before) so I might be simply wrong in here. I still find it weird that this change is made at the translation level, though, if it's a standard shouldn't it be changed in the source? A link to relevant documentation would help a bit :)

Reply:
It’s made at the translation level because quotation marks vary across languages (and locales); for example, `“ ”` for en_US, `‘ ’` for en_GB, `„ “` for de_DE, `„ ”` for ro, `» «` for sl, etc.

Reply:
Ok then